app/controllers/newflow/login_signup_controller.rb

```diff
@@ -80,6 +80,9 @@ def educator_complete
           redirect_to signup_done_path
         },
         failure: lambda {
+          @book_subjects = book_data.subjects
```

Comment:
is book_data guaranteed to not be nil?

Reply:
Not really but that is something I am working on improving in a separate PR: https://github.com/openstax/accounts/pull/840